my-app/src/pages/events/events.component.js

```diff
@@ -31,9 +36,24 @@ function EventsListComponent(props) {
               </p>
             </div>
             <p>{event.description}</p>
-            <div className={styles.button}>
-              {event.donated && <Button text="Пожертвувати" />}
-            </div>
+            <div className={styles.progressButton}>
+              <div className={styles.progress}>
+                  <Progress type="circle" strokeColor={{
+                    '0%': '#e9e210',
+                    '100%': '#e96e10',
+                    }}
+                    width={100}
+                    percent={event.currentAmount=100}  // '=100' only for test
```

Comment:
Please use totalAmount field from the database to count the total.

Reply:
Yes, please add the currentAmount field to the database and use appropriate fields to get and calculate the amount for the progress bar.